### PR TITLE
feat(cli): adds lql query data sources

### PR DESCRIPTION
**Pull request by hazedav**
## Summary

Lacework is introducing APIv2 endpoints as it pertains to LQL Query Data Sources.  These endpoints will allow us to list details for all available data sources via `GET api/v2/Datasources` as well as to list details of an individual Datasource in a RESTful manner via `GET api/v2/Datasources/<datasource_id>`.

We will introduce 2 API methods:
- `V2.Datasources.List()`
- `V2.Datasources.Get(id string)`

These will be wired to 2 CLI sub-commands of `lacework query`:
- `lacework query list-sources`
- `lacework query show-source <datasource_id>`

These will also be aliased to:
- `lacework query sources`
- `lacework query describe <datasource_id>`

These commands are currently hidden as the endpoint is not quite available yet.
The integration tests are disabled as the endpoint is not quite available yet.

## How did you test this change?
This change is tested via a set of unit tests `lql_sources_test.go` and integration tests `lql_sources_test.go`.

## Issue
https://lacework.atlassian.net/browse/ALLY-694

### CODE REVIEW

File: api/datasources_test.go [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+	)
+	defer fakeServer.Close()
+
+	c, err := api.NewClient("test",
+		api.WithToken("TOKEN"),
+		api.WithURL(fakeServer.URL()),
+	)
+	assert.Nil(t, err)
+
+	_, err = c.V2.Datasources.List()
```
Comment (afiune) [inline api/datasources_test.go:64]: could we test the results from these functions?

File: api/datasources_test.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	)
+	defer fakeServer.Close()
+
+	c, err := api.NewClient("test",
+		api.WithToken("TOKEN"),
+		api.WithURL(fakeServer.URL()),
+	)
+	assert.Nil(t, err)
+
+	_, err = c.V2.Datasources.Get("CloudTrailRawEvents")
```
Comment (afiune) [inline api/datasources_test.go:135]: Like this one 👆🏽

File: api/datasources_test.go [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
```
Comment (afiune) [inline api/datasources_test.go:207]: and this one. At least check that the response is empty.

**Review by afiune — APPROVED** [review]
All my comments are nitpicking, great work @hazedav - as always! 

tenor-219089893

File: api/datasources_test.go (hunk 1 above)
Comment (hazedav) [reply api/datasources_test.go:64]: We test the results from these functions in subsequent tests.  This test is all about ensuring the request method type.

**Pull request merged**